solidity/contracts/ViewSPV.sol

```diff
@@ -0,0 +1,116 @@
+pragma solidity ^0.5.10;
+
+/** @title ViewSPV */
+/** @author Summa (https://summa.one) */
+
+import {TypedMemView} from "./TypedMemView.sol";
+import {ViewBTC} from "./ViewBTC.sol";
+import {SafeMath} from "./SafeMath.sol";
+
+
+library ViewSPV {
+    using TypedMemView for bytes;
+    using TypedMemView for bytes29;
+    using ViewBTC for bytes29;
```

Comment:
I'm not very familiar with Solidity, so I might not be understanding this correctly...  But do we need to use `TypedMemView` and `ViewBTC` for bytes29?